app/apollo/resolvers/channel.js

```diff
@@ -145,7 +145,7 @@ const channelResolvers = {
         const channelFilter = channelName ? { name: channelName, org_id } : { uuid: channelUuid, org_id } ;
         const channel = await models.Channel.findOne(channelFilter);
         if(!channel){
-          throw new NotFoundError(context.req.t('Could not find the channel with uuid/name {{channelUuid}}/channelName.', {'channelUuid':channelUuid}), context);
+          throw new NotFoundError(context.req.t('Could not find the configuration channel with uuid/name {{channel_uuid}}/channelName.', {'channel_uuid':channel_uuid}), context);
```

Comment:
this should be 
         ` throw new NotFoundError(context.req.t('Could not find the configuration channel with uuid/name {{channelUuid}}/channelName.', {'channelUuid':channelUuid}), context);`

Reply:
modified.